tests/pkg_tweetnacl/Makefile

```diff
@@ -0,0 +1,25 @@
+APPLICATION = tweetnacl_test
+
+# If no BOARD is found in the environment, use this default:
```

Comment:
Personally, I would remove all the comments except the tweetnacl specific ones here, but that's a matter of taste.